routers/users-router.js

```diff
@@ -0,0 +1,108 @@
+express = require('express');
+
+const db = require('../data/helpers/userDb.js');
+
+const router = express.Router();
+
+//Custom middleware
+const bigName = (req, res, next) => {
```

Comment:
I would do a check if there even is a name first or else you get a crash if provide something like an age instead